CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ _**Note:** This is in reverse chronological order, so newer entries are added to
 
 ## Swift 5.7
 
+* [SE-0326][]:
+
+Parameter and result types can now be inferred from from the body of multi-statement closures.
```

Comment:
```suggestion
Parameter and result types can now be inferred from the body of multi-statement closures.
```

Reply:
These lines should also be indented by two spaces, to become part of the list item.

(The last few changelog entries have the same problem.)

Reply:
@benrimmington I updated the style for others too:)